src/test/java/com/dsrm/dsrmbackend/admin/AdminReservationControllerTests.java

```diff
@@ -90,4 +91,14 @@ void tryToAddInvalidReservation() throws Exception {
                         "to must not be null"))));
     }
 
+
+    @Test
+    @Transactional
+    void deleteValidReservation() throws Exception{
+        this.mockMvc.perform(delete("/admin/reservations/1").contentType(MediaType.APPLICATION_JSON))
+                .andExpect(status().isNoContent());
```

Comment:
It would be great to check if reservation was deleted

Reply:
functionality exists but github didn't see it